server/api.go

```diff
@@ -444,7 +451,14 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if whitelistSize >= a.p.getConfiguration().ConnectedUsersAllowed {
+	invitedUser, err := a.p.store.GetInvitedUser(mmUserID)
+	if err != nil {
+		a.p.API.LogWarn("Error in getting invited user", "error", err.Error())
+		http.Error(w, "Something went wrong", http.StatusInternalServerError)
+		return
+	}
+
+	if whitelistSize >= a.p.getConfiguration().ConnectedUsersAllowed && invitedUser == nil {
```

Comment:
@lieut-data I think we need this change to ensure non-invited new connections don't "jump the line" during the last batch of invites.

e.g.: max 100, 90 connected, 10 pending invites—then 5 non-invited users connect, then the 10 invites connect—now we're at 105 connected


```suggestion
	if (whitelistSize + invitedSize) >= a.p.getConfiguration().ConnectedUsersAllowed && invitedUser == nil {
```

Reply:
That makes sense to me!